kernel/comps/logger/src/aster_logger.rs

```diff
@@ -14,15 +14,17 @@ impl log::Log for AsterLogger {
     }
 
     fn log(&self, record: &Record) {
-        let timestamp = Jiffies::elapsed().as_duration().as_secs_f64();
-        print_logs(record, timestamp);
+        let millis = Jiffies::elapsed().as_duration().as_millis();
+        let secs = (millis / 1000) as u64;
+        let millis = (millis % 1000) as u64;
+        print_logs(record, secs, millis);
     }
 
     fn flush(&self) {}
 }
 
 #[cfg(feature = "log_color")]
-fn print_logs(record: &Record, timestamp: f64) {
+fn print_logs(record: &Record, secs: u64, millis: u64) {
```

Comment:
I understand that you want to avoid using floating point numbers. But the new signature is not quite clean. Instead let it take a `Duration`.

```rust
fn print_logs(record: &Record, timestamp: &Duration) {
```